src/coreclr/jit/codegenxarch.cpp

```diff
@@ -5783,9 +5783,9 @@ void CodeGen::genJmpMethod(GenTree* jmp)
 
             if (varTypeIsFloating(loadType))
             {
-                intArgReg       = compiler->getCallArgIntRegister(argReg);
-                instruction ins = ins_CopyFloatToInt(loadType, TYP_LONG);
-                inst_RV_RV(ins, argReg, intArgReg, loadType);
+                intArgReg = compiler->getCallArgIntRegister(argReg);
+                // there was an error here, src xmm was expected as the first arg, but was passed as second.
```

Comment:
I will delete this comment before the merge, but this is a semantic change, please review it carefully.